binaries/convert_image_to_tensor.cc

```diff
@@ -145,7 +145,7 @@ std::vector<float> convertOneImage(std::string& filename) {
   std::cout << "Converting " << filename << std::endl;
   // Load image
   cv::Mat img = cv::imread(
-      filename, FLAGS_color ? CV_LOAD_IMAGE_COLOR : CV_LOAD_IMAGE_GRAYSCALE);
+      filename, FLAGS_color ? cv::IMREAD_COLOR : cv::IMREAD_GRAYSCALE);
```

Comment:
For OpenCV 4.0.

Reply:
In that case you'll need to do some flagging, since this will make it incompatible with OpenCV 2.4

Reply:
Actually, I take that back - looks like we've already dropped support in https://github.com/pytorch/pytorch/pull/9966 so this is fine. Thanks for the clarification.

Reply:
Sure, I'll fix that anyway.

Reply:
Fixed it. 
The only thing remaining is to refactor ImageInputOp to avoid unresolved_external problem when building in debug. 
How can I test refactored operator's code before submitting?

Reply:
Figured out, that there is a python test for this op, which is being skipped by auto-testing system.
I'll check that test on my own at least on my PC.